packages/svelte/src/internal/client/dom/blocks/slot.js

```diff
@@ -13,6 +15,17 @@ export function slot(anchor, $$props, name, slot_props, fallback_fn) {
 	}
 
 	var slot_fn = $$props.$$slots?.[name];
+
+	if (slot_fn === undefined) {
+		var possible_snippet = $$props[name];
+		if (
+			typeof possible_snippet === 'function' &&
+			(!DEV || validated_snippets.has(possible_snippet))
```

Comment:
That feels brittle to me - it means things work subtly different in prod, which could break things.